src/app/components/shared/cms/cms.component.ts

```diff
@@ -48,6 +48,8 @@ export class CmsComponent extends WithUnsubscribe() implements OnInit {
       .pipe(takeUntil(this.unsubscribe))
       .subscribe(
         (data) => {
+          // TODO Validate if this is needed?
```

Comment:
I have confirmed the server will sanitize rendered markdown blobs

Reply:
Currently, this is needed so that some non-approved html elements can be applied to CMS pages. Namely, the home component uses the `<style>` element to apply styling (although that is no longer needed because of #421).

Reply:
#421 had an update which now requires access to the `<style>` element again. I wont be able to remove the sanitization applied to the CMS without breaking the home page.